CMakeLists.txt

```diff
@@ -34,6 +34,7 @@
 
 cmake_minimum_required(VERSION 2.8.12)
 project(rocksdb)
+enable_language(C ASM)
```

Comment:
why not enable C++? as most part of rocksdb is written in this language =)

please note,  the 2nd param of `enable_language()` is ignored, see https://cmake.org/cmake/help/v3.0/command/enable_language.html. might want to call it multiple times. or just use `project()` command.